packages/block-editor/src/components/list-view/style.scss

```diff
@@ -379,8 +378,10 @@
 		}
 	}
 
-	.block-editor-list-view-block-select-button__label-wrapper {
-		min-width: 120px;
```

Comment:
I could be wrong, but it looks like the removal of the min-width can result in very truncated / unreadable labels at the deepest nesting level. This is a bit of a contrived example, but in the following, there's a deeply nested Group block with a custom html anchor:

<img width="347" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/ff3bb3c5-a6fe-465d-9f6f-ecde39865734">

At this nesting level and with the lock icon, the icon winds up overlapping the settings menu button a bit, too:

<img width="345" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/fc9f44e3-7fba-4824-b105-3f6b555e9014">

Reply:
I pushed some spacing tweaks to try based on discussion in the issue. Here's what I now see at the deepest nesting level:

<img width="349" alt="Screenshot 2024-04-18 at 13 54 43" src="https://github.com/WordPress/gutenberg/assets/846565/1f6b6b49-d507-466b-80c6-378d1e18ecbc">

I greatly appreciate the lack of horizontal scrolling. The truncation is still a bit aggressive when an anchor is set, but it might be passable? 

Not a strong feeling, happy to re-instate the `min-width`.

Reply:
That's feeling much better to me. It's a difficult balance to strike, but for my tastes, the absence of having to scroll horizontally beats out the desire for a large minimum width of the label. I think this would be good to try, and it sounds like it'd be a fairly simple follow-up to re-instate the `min-width` if folks find it confusing with this level of truncation 👍